docs/src/locators.md

```diff
@@ -221,106 +347,101 @@ await page.GetByTestId("product-item").Filter(new() { HasText = "Playwright Book
+```js
+await page.getByAltText('playwright logo').click();
 ```
 
 ```java
-page.getByRole("button", new Page.GetByRoleOptions().setName(Pattern.compile("submit", Pattern.CASE_INSENSITIVE))).click();
-
-page.getByRole("checkbox", new Page.GetByRoleOptions().setChecked(true).setName("Check me"))).check();
+page.getByAltText("playwright logo").click();
 ```
 
-```csharp
-await page.GetByRole("button", new() { Name = new Regex("submit", RegexOptions.IgnoreCase) }).ClickAsync();
-
-await page.GetByRole("checkbox", new() { Checked = true, Name = "Check me" }).CheckAsync();
+```python async
+await page.get_by_alt_text("playwright logo").click()
 ```
 
-Role locators follow W3C specificaitons for [ARIA role](https://www.w3.org/TR/wai-aria-1.2/#roles), [ARIA attributes](https://www.w3.org/TR/wai-aria-1.2/#aria-attributes) and [accessible name](https://w3c.github.io/accname/#dfn-accessible-name).
+```python sync
+page.get_by_alt_text("playwright logo").click()
+```
 
-Note that role locators **do not replace** accessibility audits and conformance tests, but rather give early feedback about the ARIA guidelines.
+```csharp
+await page.getByAltText("playwright logo").click();
+```
+### Locate by title with getByTitle
+[`method: Page.getByTitle`]
 
-### Define explicit contract and use [`method: Page.getByTestId`]
+Locate an element with a matching title attribute using [`method: Page.getByTitle`].
 
-User-facing attributes like text or accessible name can change over time. In this case it is convenient to define explicit test ids.
+For example, consider the following DOM structure.
 
 ```html
-<button data-testid="directions">Itinéraire</button>
+<span title='Close'></span>
 ```
 
+You can fill the input after locating it by the label text:
+
 ```js
-await page.getByTestId('directions').click();
+await frame.getByTitle('Close').click();
 ```
 
 ```java
-page.getByTestId("directions").click();
+page.getByTitle("Close").click();
 ```
 
 ```python async
-await page.get_by_test_id('directions').click()
+await page.get_by_label("Close").click()
 ```
 
 ```python sync
-page.get_by_test_id('directions').click()
+page.get_by_label("Close").click()
 ```
 
 ```csharp
-await page.GetByTestId("directions").ClickAsync();
+await page.GetByTitle("Close").Click();
 ```
 
-By default, [`method: Page.getByTestId`] will locate elements baed on the `data-testid` attribute, but you can configure it in your test config or calling [`method: Selectors.setTestIdAttribute`].
-
-### Locate by label text with [`method: Page.getByLabel`]
+### Define explicit contract and use getByTestId
 
-Most form controls usually have dedicated labels that could be conveniently used to interact with the form. In this case, you can locate the control by its associated label.
-
-For example, consider the following DOM structure.
+User-facing attributes like text or accessible name can change over time. In this case it is convenient to define explicit test ids using [`method: Page.getByTestId`].
```

Comment:
You are not defining test ids using this method, you only query using this method. Perhaps we should expand on "how to define test ids" to make this clear?